pkg/models/trb_guidance_letter_recommendation.go

```diff
@@ -11,10 +14,73 @@ import (
-	PositionInLetter null.Int       `json:"positionInLetter" db:"position_in_letter"` // 0-based indexing
-	DeletedAt        *time.Time     `json:"deletedAt" db:"deleted_at"`
+	TRBRequestID     uuid.UUID                               `json:"trbRequestId" db:"trb_request_id"`
+	Title            string                                  `json:"title" db:"title"`
+	Recommendation   HTML                                    `json:"recommendation" db:"recommendation"`
+	Links            pq.StringArray                          `json:"links" db:"links"`
+	PositionInLetter null.Int                                `json:"positionInLetter" db:"position_in_letter"` // 0-based indexing
+	DeletedAt        *time.Time                              `json:"deletedAt" db:"deleted_at"`
+	Category         TRBGuidanceLetterRecommendationCategory `json:"category" db:"category"`
 }
+
+type TRBGuidanceLetterRecommendationCategory string
+
+const (
+	TRBGuidanceLetterRecommendationCategoryRequirement    TRBGuidanceLetterRecommendationCategory = "REQUIREMENT"
+	TRBGuidanceLetterRecommendationCategoryRecommendation TRBGuidanceLetterRecommendationCategory = "RECOMMENDATION"
+	TRBGuidanceLetterRecommendationCategoryConsideration  TRBGuidanceLetterRecommendationCategory = "CONSIDERATION"
+	TRBGuidanceLetterRecommendationCategoryUncategorized  TRBGuidanceLetterRecommendationCategory = "UNCATEGORIZED"
+)
+
+var AllTRBGuidanceLetterRecommendationCategory = []TRBGuidanceLetterRecommendationCategory{
+	TRBGuidanceLetterRecommendationCategoryRequirement,
+	TRBGuidanceLetterRecommendationCategoryRecommendation,
+	TRBGuidanceLetterRecommendationCategoryConsideration,
+	TRBGuidanceLetterRecommendationCategoryUncategorized,
+}
+
+func (e TRBGuidanceLetterRecommendationCategory) IsValid() bool {
+	switch e {
+	case TRBGuidanceLetterRecommendationCategoryRequirement, TRBGuidanceLetterRecommendationCategoryRecommendation, TRBGuidanceLetterRecommendationCategoryConsideration, TRBGuidanceLetterRecommendationCategoryUncategorized:
+		return true
+	}
+	return false
+}
+
+func (e TRBGuidanceLetterRecommendationCategory) String() string {
+	return string(e)
+}
+
+// UnmarshalGQL is a custom implementation of what gqlgen would normally generate
+// see Note [gql enums]
+func (e *TRBGuidanceLetterRecommendationCategory) UnmarshalGQL(v interface{}) error {
+	switch t := v.(type) {
+	case TRBGuidanceLetterRecommendationCategory:
+		*e = t
+		if !e.IsValid() {
+			return fmt.Errorf("%s is not a valid TRBGuidanceLetterRecommendationCategory", t.String())
+		}
+
+		return nil
+
+	case string:
+		*e = TRBGuidanceLetterRecommendationCategory(t)
+		if !e.IsValid() {
+			return fmt.Errorf("%s is not a valid TRBGuidanceLetterRecommendationCategory", t)
+		}
+
+		return nil
+	}
+
+	return fmt.Errorf("could not parse %v as a TRBGuidanceLetterRecommendationCategory", v)
+}
+
+func (e TRBGuidanceLetterRecommendationCategory) MarshalGQL(w io.Writer) {
+	fmt.Fprint(w, strconv.Quote(e.String()))
+}
```

Comment:
Are these methods necessary or will the update call work without them? I'd rather not be in the business of re-implementing library code if we don't need it.

Reply:
will check

Reply:
removed `MarshalGQL` - it does not appear to be needed

Reply:
I'd also probably remove `UnmarshalGQL` if we're not calling it anywhere. I think you have to have both `MarshalGQL` and `UnmarshalGQL` to implement the interface that gqlgen looks for (so removing one will make it not satisfy it anymore) but I'd say removing both is probably best, since it's also currently unused

Reply:
removed! thanks Clay, appreciate you pointing that out!